### DISCUSSION TITLE
Add color palette

### DISCUSSION THREAD

**Discussion by ghost** [Orama-Interactive/Pixelorama#745] [Ideas]
Necessary for finer work with colors

**Comment by Variable-ind** [comment]
Isn't there already a color palette in pixelorama?

**Reply by ghost** [reply]
@Variable-ind there is but you cant select color u need using RGBA. You should use defaults colors or make your custom palette

**Reply by OverloadedOrama** [reply]
@MIR2006BY You can already do that. See this documentation page for more information. https://www.oramainteractive.com/Pixelorama-Docs/user_manual/palettes
